RedditSharp/Things/VotableThing.cs

```diff
@@ -158,5 +169,46 @@ public void ClearVote()
             var response = request.GetResponse();
             var data = WebAgent.GetResponseString(response.GetResponseStream());
         }
+
+        public void Report(ReportType reportType)
+        {
+            Report(reportType, "");
+        }
+
+        public void Report(ReportType reportType, string otherReason)
```

Comment:
Wasn't sure if you wanted it to be compatible with earlier .NET's that don't have default parameters.